ocaml/tests/test_repository.ml

```diff
@@ -24,16 +24,17 @@ let test_introduce_duplicate_name () =
   let source_url = "https://repo-src.example.com" in
```

Comment:
Would be clear to rename `test_introduce_duplicate_name` to `test_introduce_duplicate_remote_repo`